annotations.py

```diff
@@ -22,9 +22,13 @@
 from calibre_plugins.annotations.common_utils import Logger
 from calibre_plugins.annotations.config import plugin_prefs
 
+try:
+    from PyQt5.QtGui import QColor
+except ImportError:
+    from PyQt4.QtGui import QColor
+
 COLOR_MAP = {
                   'Blue': {'bg': '#b1ccf3', 'fg': 'black'},
-               'Default': {'bg': 'transparent', 'fg': 'black'},
```

Comment:
Are you sure this removed entry isn't used anywhere else?

Reply:
Yes